.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - '7.1'
+  - '7.3'
```

Comment:
I take it going to php 7.4 is a step too far? (Also: Hey Laurens!)

Reply:
Hi Thijs,
You are right! we should test also in PHP 7.4